docs/UserGuide.md

```diff
@@ -361,7 +361,15 @@ to find your schedule for a specific day of the week.
 
 ### Clearing all entries : `clear`
 
-Clears all entries from the address book.
+Clears all data from the data file of UGTeach.
+
+<box type="warning" header ="##### Caution">
+This action is not undoable and your deleted data cannot be recovered. You should save a copy of the data file before trying this command.
+
+* Locate to the folder to save `ugteach.jar` and go to the folder `data`.
```

Comment:
If you wanna keep, I want to suggest that:

- To locate the data file, go to the folder `data`, and find a file called `ugteach.json`.
- You can then duplicate the data file and save it somewhere else.